recipes-extended/xen/files/0001-convert-block-tap-for-shared.patch

```diff
@@ -511,6 +511,8 @@ xenmgr to allow tapdisk to unlock encrypted VHDs.
 +
 +    if [ "$mode" = "r" ]; then
 +        tap_opts="-R"
++    else
++        tap_opts="-W"
```

Comment:
This would cover the non-measured, HVM (Windows/Linux) use case?

Reply:
Yes.  read-only/read-write is independent of measured, but measured only measures read-only disks.